docs/cmdlet-example/command-line-simple-example.md

```diff
@@ -0,0 +1,135 @@
+# Creating a Simple Cross-Platform Binary Module with the Dotnet CLI
+
+This example shows how you can create a simple, binary module using the dotnet
+CLI (command line interface).  For this example, you can use any OS that the
+.NET Core 2.0 SDK installs on.  The Windows for Subsystem Linux console works
```

Comment:
Should be `Windows Subsystem for Linux` not `Windows for Subsystem Linux`

Reply:
fixed